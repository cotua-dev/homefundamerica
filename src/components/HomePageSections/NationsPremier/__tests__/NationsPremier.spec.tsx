import React from 'react';
import { render, RenderResult } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import { default as NationsPremier } from '..';

describe('NationsPremier component', (): void => {
    test('renders', (): void => {
        const { container }: RenderResult = render(<NationsPremier/>);

        expect(container.firstChild).toMatchSnapshot();
    });
});
