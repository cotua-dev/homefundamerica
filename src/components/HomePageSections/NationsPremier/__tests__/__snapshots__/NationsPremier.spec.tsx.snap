// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NationsPremier component renders 1`] = `
<section>
  <div>
    <h1>
      Nation's Premier
    </h1>
    <h1>
      Mortgage Lender
    </h1>
    <h3>
      Mortgages for Self-Employed Borrowers and Investors
    </h3>
    <p>
      We have made the mortgage process fast and easy to reduce costs, unnecessary steps, and time-wasting appointments. So what are you waiting for? Click the button below to get started!
    </p>
    <a
      href="/conventional"
    >
      <button
        type="button"
      >
        Get Started
      </button>
    </a>
    <a
      href="tel:8888186393"
    >
      (888) 818-6393
    </a>
  </div>
</section>
`;
